src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ExcessHet.java

```diff
@@ -193,20 +190,19 @@ static double exactTest(final int hetCount, final int refCount, final int homCou
             currHomC = currHomC - 1;
         }
 
-        final double rightPval = probs[hetCount] / (2.0 * mysum);
+        final double rightPval = Math.min(1., probs[hetCount] / mysum);
         //Check if we observed the highest possible number of hets
         if (hetCount == rareCopies) {
             return rightPval;
         }
-        return rightPval + StatUtils.sum(Arrays.copyOfRange(probs, hetCount + 1, probs.length)) / mysum;
+        return Math.min(1., rightPval + StatUtils.sum(Arrays.copyOfRange(probs, hetCount + 1, probs.length)) / mysum);
```

Comment:
BTW, seems like we should use something from QualityUtils for converting this to Phred scale in the calculateEH method, but will let the reviewer decide. Not sure if we will want to e.g. introduce any caps.